README.md

```diff
@@ -26,6 +26,10 @@ Blog App is a fully functional website that will show the list of posts and empo
 
 - - Type `rails db:migrate` to run migration files
 
+**To run development environment and build tailwindcss files**
```

Comment:
### Optional
- [ ] It would be better if we implement validation for our model, currently a user can enter empty data which is costly for our database